e2e_test/e2e_test.go

```diff
@@ -36,3 +37,44 @@ func TestSendCelo(t *testing.T) {
 	err = network.AwaitTransactions(ctx, tx)
 	require.NoError(t, err)
 }
+
+func TestSingleNodeManyTxs(t *testing.T) {
+	rounds := 100
+	accounts := test.Accounts(1)
```

Comment:
With only 1 account we only create 1 node and so at that point we are not testing a network and not executing most of the consensus code. So unless we specifically want to avoid executing that code I suggest a minimum of 3 accounts.

Reply:
yes, this is intentional. I think having just one test with a single node will probably make it easier to debug problems in the future (as in, if something fails on the tests with multiple nodes but work in this one, or any other case)